features/images.feature

```diff
@@ -0,0 +1,40 @@
+@images
+Feature: OpenStack Image management
+  As an administrator
+  I want to make sure that OpenStack Image management component
+  is working correctly
+
+  @create_kvm
+  Scenario: Creation of KVM image
+    Given KVM source image exists
+    And glance image does not exist
+    When I create new glance image based on jeos
+    Then this image has non-empty ID
+    And its ID can be used to show the image info
+    And the status of the image is active
+    When I delete the image
```

Comment:
I'm not sure. All scenarios are creating different images, so I'm not sure why should I have one scenario for deleting all of them.

And also, I like it more when the situation is back to original state after executing each scenario, not just after running full feature.

Background is nice, yes, e.g. for testing something that is really shared (and for 2 creating scenarios, they have different image id and name) and I wouldn't expect it to delete the image if it exist